### PR TITLE
transitioning AIM position add to generic url TM-3258

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: 
- [API PR ](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/700)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/295)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Dual Merge: 
-- [API PR ](linkToAPI_PR)
```
Comment (scott062) [inline .github/pull_request_template.md:2]: 🙄

File: src/Components/Agenda/AgendaItemTimeline/AgendaItemTimeline.jsx [hunk 2]
```diff
@@ -43,22 +43,18 @@ const AgendaItemTimeline = ({ unitedLoading, setParentLoadingState }) => {
...
-          org: get(pos, 'position.organization').match(regex)[1],
-          // org: get(pos, 'position.organization'),
+          pos_title: get(pos, 'title'),
+          pos_num: get(pos, 'position_number'),
+          org: get(pos, 'organization'),
           eta: '2019-05-05T00:00:00.000Z',
           ted: '2020-07-05T00:00:00.000Z',
-          tod: get(pos, 'position.tour_of_duty'),
-          grade: get(pos, 'position.grade'),
+          tod: get('position.tour_of_duty'),
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemTimeline/AgendaItemTimeline.jsx:56]: typo?

Comment (elizabeth-jimenez) [reply src/Components/Agenda/AgendaItemTimeline/AgendaItemTimeline.jsx]: yesss - updated!

**Comment by scott062** [comment]
We discussed the UI bug with the didmount functionality issue - going to merge for now, so it can be addressed in separate PRs. If you make additional requests, the page loads and the functionality works. Simply a loading state issue we need to look into more

**Review by scott062 — APPROVED** [review]
🎸

File: .github/pull_request_template.md (hunk 1 above)
Comment (scott062) [reply .github/pull_request_template.md:2]: Just funny thats been there so long

**Pull request merged**